modules/installation-guide/partials/proc_mounting-a-secret-or-a-configmap-as-a-file-into-a-container.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-container
+
+[id="mounting-a-secret-or-a-configmap-as-a-file-into-a-container_{context}"]
+= Mounting a Secret or a ConfigMap as a file into a {prod-short} container
+
+.Prerequisites
+
+* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
+
+.Procedure
+
+. Create a new {platforms-name} Secret or a ConfigMap in the {platforms-name} {orch-namespace} where a {prod-short} is deployed. The labels of the object that is about to be created must match the set of labels:
++
+* `app.kubernetes.io/part-of: che.eclipse.org`
+* `app.kubernetes.io/component: <DEPLOYMENT_NAME>-<OBJECT_KIND>`
++
+* The `<DEPLOYMENT_NAME>` coresponds to the one following deployments:
+
+** `postgres`
+** `keycloak`
+** `devfile-registry`
+** `plugin-registry`
+** `{prod-deployment}` 
++
+and
++
+* `<OBJECT_KIND>` is either:
++
+** `secret`
++
+or
++ 
+** `configmap`
+
+// The following content is downstream friendly
+
+.Example:
+====
+[source,yaml,subs="+attributes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: custom-data
+  labels:
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: {prod-deployment}-secret
+...
+----
+or
+[source,yaml,subs="+attributes"]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: custom-data
+  labels:
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: {prod-deployment}-configmap
+...
+----
+====
+
+Annotations must indicate that the given object is mounted as a file. Configure the annotation values:
+
+* `che.eclipse.org/mount-as: file` - to indicate that a object is mounted as a file
+* `che.eclipse.org/mount-path: _<FOO_ENV>_` - to provide a required mount path
+
+.Example:
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: custom-data
+  annotations:
+    che.eclipse.org/mount-path: /data
+    che.eclipse.org/mount-as: file
```

Comment:
it's not critical but above the order is different, I think it makes sense to align them